### PR TITLE
Feature/iii 3456 card component

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Added

- pub-card component
- pub-card story

---

Ticket: https://jira.uitdatabank.be/browse/III-3456

### CODE REVIEW

File: publiq-ui/pub-card.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+<Canvas>
+  <Story name="WithText">{TemplateCardWithCardText.bind({})}</Story>
+</Canvas>
+
+```javascript
+<pub-card>
+  Card content
+  <pub-card-text>Card footer text</pub-card-text>
+</pub-card>
```
Comment (bertramakers) [inline publiq-ui/pub-card.stories.mdx:44]: What's the point of the `pub-card-text` / `b-card-text` exactly?

Comment (bertramakers) [reply publiq-ui/pub-card.stories.mdx:44]: It says "Card _footer_ text" (emphasis mine), but I see no difference with just regular text content inside `pub-card`?

Comment (simon-debruijn) [reply publiq-ui/pub-card.stories.mdx:44]: With sub-component <b-card-text>, paragraph text can be added to the card. The last <b-card-text> in the card body will have its bottom margin automatically removed (via CSS). Text within <b-card-text> can also be styled with the standard HTML tags.

Comment (simon-debruijn) [reply publiq-ui/pub-card.stories.mdx:44]: Schermafbeelding 2020-09-11 om 13 29 32

Comment (simon-debruijn) [reply publiq-ui/pub-card.stories.mdx:44]: So it are standard <p> tags I think only the last p tags margin is automatically  taken care of

Comment (bertramakers) [reply publiq-ui/pub-card.stories.mdx:44]: Aha I see! Sounds good. Maybe we could clarify the story then by changing the example to something like:
```
<pub-card>
  <pub-card-text>Paragraph 1</pub-card-text>
  <pub-card-text>Paragraph 2</pub-card-text>
  <pub-card-text>Paragraph 3 (will have its bottom margin removed)</pub-card-text>
</pub-card>
```

Comment (bertramakers) [reply publiq-ui/pub-card.stories.mdx:44]: Also maybe `pub-card-paragraph` would be clearer? 🤔 But more verbose. Just throwing the idea out there.

Comment (simon-debruijn) [reply publiq-ui/pub-card.stories.mdx:47]: changes made

**Review by bertramakers — APPROVED** [review]

**Pull request merged**